docs/release.md

```diff
@@ -12,17 +12,19 @@ When we release we do the following process:
 
 ## Release commands
 
-Tag the version
+Tag the version:
 
 ```bash
 git tag -a v0.1.0-rc.0 -m "v0.1.0-rc.0"
 git push origin v0.1.0-rc.0
 ```
 
-Run goreleaser, make sure to export your GitHub token first.
+From there, github actions should automatically create the release, as it sets
```

Comment:
In this situation the github action would run after tag push but no PR would be opened right?

Reply:
Yeah so this is a behavior of goreleaser.

Basically the flow is that once an admin pushes the tag, that tag is built and released. We would need to make sure that the the image build is also triggered for the tag, but it *should* be